grocy-chores-card.js

```diff
@@ -641,7 +669,7 @@ class GrocyChoresCard extends LitElement {
         this.show_last_tracked_by = this.config.show_last_tracked_by ?? true;
         this.filter_category = this.config.filter_category ?? null;
         this.show_category = this.config.show_category ?? true;
-        this.show_description = this.config.show_description ?? true;
+        this.show_description = this.config.show_description ?? false;
```

Comment:
set to false to maintain the existing behavior (since this flag is not currently used)